workflow/controller/operator.go

```diff
@@ -1162,11 +1182,13 @@ func (woc *wfOperationCtx) executeTemplate(nodeName string, orgTmpl wfv1.Templat
 		}
 		processedRetryParentNode, err := woc.processNodeRetries(retryParentNode, *processedTmpl.RetryStrategy)
 		if err != nil {
+			woc.log.Errorf("Error: %+v", err)
```

Comment:
I moved error log to `markNodeError`.
https://github.com/argoproj/argo/pull/1694/files#diff-fcb04129f1d32e69cca32631c6586587R1426

Reply:
This is a result of the merge, will remove these.